lib/lrama/grammar.rb

```diff
@@ -273,7 +195,7 @@ def compute_nullable
         end
 
         rule_count_2  = @rules.count {|e| e.nullable.nil? }
-        nterm_count_2 = nterms.count {|e| e.nullable.nil? }
+        nterm_count_2 = nterms.select {|e| e.nullable.nil? }.count
```

Comment:
I overlooked this change in previous review. Could you keep the original codes? This line and previous line execute same things for other sets. So that it's good to be same pattern.

Reply:
Thank you catching this. I reverted this code.